fish_title.fish

```diff
@@ -3,7 +3,19 @@
 #     set -g theme_title_display_path no
 #     set -g theme_title_use_abbreviated_path no
 
+function __bobthefish_user -S -d 'Display actual user if different from $default_user'
+  if [ "$theme_display_user" = 'yes' ]
```

Comment:
let's make this `$theme_title_display_user` so they can be controlled independently.